verticapy/plotting/_plotly/machine_learning/__init__.py

```diff
@@ -0,0 +1,16 @@
+"""
+(c)  Copyright  [2018-2023]  OpenText  or one of its
+affiliates.  Licensed  under  the   Apache  License,
+Version 2.0 (the  "License"); You  may  not use this
+file except in compliance with the License.
+
+You may obtain a copy of the License at:
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless  required  by applicable  law or  agreed to in
+writing, software  distributed  under the  License is
```

Comment:
You did not have already an __init__ file for ML?

Reply:
No. that is why I was basing it on the previous branch.